exercise2.js

```diff
@@ -95,35 +138,116 @@ if (age > 65) console.log("retired"); else {
 -add another method on your object of any action you want to perform, using some property that exists on your object, making sure to utilize the 'this' keyword.
 */
 
-//your code...
-
+const yourself = class {
+    constructor(name, age, gender, hobbies, profession, education) {
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.hobbies = hobbies;
+        this.profession = profession;
+        this.education = education;
+    }
+    learn() {
+        console.log("My name is " + this.name);
+
+    }
+    poker(chips) {
+        this.hobbies += chips;
+        console.log("Look how many chips I have " + this.hobbies);
+    }
+}
+const marcelo = new yourself("Marcelo", 26, "Male", poker = 80, "Software engineer", "Bacharel");
+marcelo.learn();
+marcelo.poker(50);
 
 /************************************************************* */
 // Problem 6: Create an object literal that represents any object in the real world that you like.  Try to come up with a realistic set of properties and methods that would accurately describe that object.  In at least one of your methods, use the this keyword to refer back to one of the properties you defined on this object literal.  
 
-//your code...
+const Dog = class {
+    constructor(breed, color, age) {
+        this.breed = breed;
+        this.color = color;
+        this.age = age;
+    }
+}
 
+Dog.prototype.ageInHumanYears = function () {
+    console.log(this.age * 7)
+};
+const dog1 = new Dog("Labrador", "Brown", 3);
+const dog2 = new Dog("Pitbull", "white", 5);
+dog1.ageInHumanYears();
+dog2.ageInHumanYears();
```

Comment:
this works, but the instructions asked you to create an object literal.  
a class and prototypical inheritanced were not required.